src/main/scala/uk/gov/nationalarchives/Processor.scala

```diff
@@ -41,22 +41,42 @@ class Processor(
       fileName: String,
       path: String,
       repType: Option[String] = None
-  ): IO[List[MetadataObject]] = {
-    val consolidatedMetadata =
-      <XIP xmlns="http://preservica.com/XIP/v7.0">
-          {
-        Seq(metadata.entityNode)
+  ): IO[List[MetadataObject]] =
+    for {
+      entitySpecificMetadata <-
+        metadata match {
+          case ioMetadata: IoMetadata =>
+            IO(
+              Seq(ioMetadata.entityNode)
+                ++ ioMetadata.representations.flatMap(representation => Seq(newlineAndIndent, representation))
+            )
+          case coMetadata: CoMetadata =>
+            IO(
+              Seq(coMetadata.entityNode)
+                ++ coMetadata.generationNodes.flatMap(generationNode => Seq(newlineAndIndent, generationNode))
```

Comment:
Do we need these new lines? Are they just for pretty printing? The reason I'm thinking maybe not is that these XML files are going to be read by the process that populates the database for the front end and almost never by a human. 
And also, if these get stored on tape in any large quantity, that's a fair bit of extra space used. 
What do you think?

Reply:
Yes, they are just for pretty printing; I did it in case someone wanted to open the files via CLI or something. It's only for the main/parent nodes, so there will only be a handful new lines unless there are a ton of metadata or identifier nodes . Do you think I should still remove it?

Reply:
Ah no, if it's only a few of the parent nodes then we should be fine. Leave it. We'll probably end up looking at it more than we think anyway.